.github/workflows/main.yml

```diff
@@ -63,13 +63,34 @@ jobs:
         run: |
           unset TORCH_CUDA_ARCH_LIST # only jit compile for current arch
           if [[ -d ./torch-extensions ]]; then rm -rf ./torch-extensions; fi
-          TORCH_EXTENSIONS_DIR=./torch-extensions pytest --durations=0 --forked --verbose tests/unit/
+          cd tests
+          TORCH_EXTENSIONS_DIR=./torch-extensions pytest --durations=0 --forked --verbose unit/
 
+  nv-transformers-v100:
+    runs-on: [self-hosted, nvidia, torch18, v100]
+    
+    steps:
+      -uses: actions/checkout@v2
+      
+      - name: environment
+        run: |
+          nvidia-smi
+          which python
+          python --version
+          which nvcc
+          nvcc --version
+          python -c "import torch; print('torch:', torch.__version__, torch)"
+          python -c "import torch; print('CUDA available:', torch.cuda.is_available())"
+      - name: Install deepspeed
+        run: |
+          pip install .[dev]
+          ds_report
       - name: HF transformers tests
         run: |
           if [[ -d ./torch-extensions ]]; then rm -rf ./torch-extensions; fi
           git clone https://github.com/huggingface/transformers
+          git rev-parse --short HEAD
           cd transformers
-          pip install -e .
+          pip install .
           pip install sacrebleu rouge-score nltk parameterized psutil datasets
```

Comment:
Let's see if this work, but alternatively we could do:
```
find examples/pytorch -name requirements.txt -exec pip install -r {} \;
```
which will be automatically finding all the dependencies. As each example has its own `requirements.txt` now.